spec/features/step_by_step_actions_spec.rb

```diff
@@ -21,11 +21,34 @@
     end
   end
 
+  context "Step by step has been submitted for 2i" do
+    before do
+      given_there_is_a_step_by_step_with_status "submitted_for_2i"
+    end
+
+    scenario "show the relevant actions to the step by step author" do
```

Comment:
I'd add a step here to match the scenario, like:

`and_I_am_the_step_by_step_author`

...

```ruby
def and_I_am_the_step_by_step_author
  @current_user = @review_requester
end
```

Reply:
Updated as suggested.